solidity/contracts/helpers/TestConverterRegistry.sol

```diff
@@ -7,8 +7,11 @@ import "../converter/ConverterRegistry.sol";
 */
 contract TestConverterRegistry is ConverterRegistry {
     IConverter public createdConverter;
+    bool public typeChangingEnabled;
 
-    constructor(IContractRegistry _registry) public ConverterRegistry(_registry) {}
+    constructor(IContractRegistry _registry) public ConverterRegistry(_registry) {
+        typeChangingEnabled = true;
```

Comment:
You can set `typeChangingEnabled` to true directly in its definition above

Reply:
True, but I figured it's a bad practice when a non-default (empty) constructor is implemented, since it is not clear when the initialization of the state-variable takes place.
Recommendation?

Reply:
I'd just initialize it in the definition and leave the constructor empty

Reply:
But the constructor is not empty, it needs to call the parent's constructor.

Reply:
Empty as in `constructor(IContractRegistry _registry) public ConverterRegistry(_registry) {}`

Reply:
Fixed.